tmva/sofie/inc/TMVA/ROperator_Slice.hxx

```diff
@@ -97,13 +102,26 @@ public:
       // loop on the extra 2 or 3 or 4 inputs
       for (size_t i = 0; i < fNames.size(); ++i) {
         if (!fNames[i].empty()) { 
-         std::cout << " i " << i << " getting data for tensor " << fNames[i] << std::endl;
+         // std::cout << " i " << i << " getting data for tensor " << fNames[i] << std::endl;
```

Comment:
we can maybe remove this line, since it is no longer required.

Reply:
I will remove it in a separate PR, since Joanna has left